src/clj/rems/handler.clj

```diff
@@ -14,24 +13,6 @@
             [rems.util :refer [never-match-route]]
             [ring.util.response :refer [file-response]]))
 
-(defn init
-  "init will be called once when
-   app is deployed as a servlet on
-   an app server such as Tomcat
-   put any initialization code here"
-  []
-  (doseq [component (:started (mount/start))]
-    (log/info component "started")))
```

Comment:
Do we want to add some new logging for something? Maybe not for these?

Reply:
standalone rems logs similar stuff:
```
18:31,258 [main - - -] INFO  rems.standalone - #'rems.config/env started
18:31,258 [main - - -] INFO  rems.standalone - #'rems.config/oidc-configuration started
18:31,258 [main - - -] INFO  rems.standalone - #'rems.db.core/*db* started
18:31,258 [main - - -] INFO  rems.standalone - #'rems.locales/translations started
18:31,258 [main - - -] INFO  rems.standalone - #'rems.db.applications/all-applications-cache started
18:31,258 [main - - -] INFO  rems.standalone - #'rems.db.applications/all-applications-cache-reloader started
18:31,258 [main - - -] INFO  rems.standalone - #'rems.application.search/search-index started
18:31,258 [main - - -] INFO  rems.standalone - #'rems.jwt/jwk-provider started
18:31,258 [main - - -] INFO  rems.standalone - #'rems.db.entitlements/entitlement-poller started
18:31,258 [main - - -] INFO  rems.standalone - #'rems.email.core/email-poller started
18:31,258 [main - - -] INFO  rems.standalone - #'rems.event-notification/event-notification-poller started
18:31,258 [main - - -] INFO  rems.standalone - #'rems.middleware/session-store started
18:31,258 [main - - -] INFO  rems.standalone - #'rems.css.styles/rendered-css-files started
18:31,258 [main - - -] INFO  rems.standalone - #'rems.home/memoized-render-css started
18:31,259 [main - - -] INFO  rems.standalone - #'rems.handler/handler started
18:31,264 [main - - -] INFO  rems.standalone - #'rems.standalone/http-server started
18:31,264 [main - - -] INFO  rems.standalone - #'rems.standalone/repl-server started
```